data/transactions/transaction.go

```diff
@@ -412,6 +412,18 @@ func (tx Transaction) TxAmount() basics.MicroAlgos {
 	}
 }
 
+// GetReceiverAddress returns the address of the receiver. If the transaction has no receiver, it returns the empty address.
+func (tx Transaction) GetReceiverAddress() basics.Address {
+	switch tx.Type {
+	case protocol.PaymentTx:
+		return tx.PaymentTxnFields.Receiver
+	case protocol.AssetTransferTx:
```

Comment:
should the 'Receiver' be the clawback address in case of clawback transaction?

Reply:
I think that only the sender changes. unless I'm missing something.